README.md

```diff
@@ -327,142 +324,143 @@ CANCEL_ERROR     'CANCEL_ERROR'     ---           Request has been cancelled. On
 
 Which problem is chosen will be picked by walking down the list.
 
-
 # Feedback
 
-Bugs?  Comments?  Features?  PRs and Issues happily welcomed!
-
+Bugs? Comments? Features? PRs and Issues happily welcomed!
 
 # Release Notes
 
 ### 1.0.0 - October 9, 2018
```

Comment:
Let's remove all of these entries and link to the releases page: 

Release notes are now located [here](https://github.com/infinitered/apisauce/releases).

Reply:
Yeah, probably a good idea. I'm going to copy all of the notes over though, since the exisiting release page is just tags with no notes. 

Does CircleCI/Semantic release have a way to add notes to releases? Or does that just get done manually in GH after the fact?